en/docs/develop/api-policies/create-a-policy.md

```diff
@@ -0,0 +1,291 @@
+public function getOrDefault(string name, anydata default) returns anydata;
+
+
+# Checks whether a mapping exists for the specified key.
+public function hasKey(string name) returns boolean;
+
+
+# Returns the `mediation:Context` instance which captured the initial contextual information of the resource,
+# before the mediation flow was invoked. Calling this on an original `mediation:Context` object will return itself.
+public function originalContext() returns Context;
+
+
+# The HTTP method of the resource method
+public function httpMethod() returns string;
+
+
+# Retrieves an instance of `mediation:ResourcePath` which is an API for contextual information on the resource path
+# of this resource. It also contains methods for modifying the resource path as the user sees fit. This resource
+# path is the same path used by the mediation service for deriving the backend endpoint's resource to invoke.
+# Therefore, the default behavior of the mediation service is to invoke a resource in the backend endpoint which
+# has the same relative resource path as the corresponding mediation service resource.
+public function resourcePath() returns ResourcePath;
+
+
+# Sets the given `mediation:ResourcePath` instance as the resource path of this context.
+public function setResourcePath(ResourcePath path);
+
+
+# Adds a mapping between a path param name and a resolved value for it. There need not be a path parameter in the
+# resource path by the name specified in `name` for one to use this method. On its own, the path param values have
+# no bearing on the resource path.
+public function addPathParamValue(string name, PathParamValue value);
+
+
+# Returns the collection of resolved values for the path parameters in this particular context, mapped
+# by the parameter name.
+public function resolvedPathParams() returns map<PathParamValue> & readonly;
+
+
+# Removes the resolved path parameter value which maps to the specified name.
+public function removePathParamValue(string name);
+
+
+# Adds a query parameter to the request to be sent to the backend. If there is already a query parameter by
+# with the same name, the new value will be appended to it, making it an array.
+public function addQueryParam(string name, string value);
+
+
+# Removes the specified query parameter from the request. If the value of the parameter is an array, the whole
+# array will be removed.
+public function removeQueryParam(string name);
+
+
+# Retrieves a map of all the query params in the current request context. The returned map is a read only snapshot
+# of the map of query parameters in the context at the time this method was called.
+public function queryParams() returns map<string[]> & readonly;
+```
+
+### Keywords
+
+The following key words are used in the Ballerina.toml file for mediation policies to work:
+
+- **choreo-apim-mediation-policy**: Mandatory keyword which identifies this package is a mediation policy type.
```

Comment:
Suggestion:
"This keyword is required to identify that this package is a mediation policy type."

Reply:
changed

Reply:
The current version is "This keyword is a mandatory keyword that is required to identify that the package is a mediation policy type."

The repetition of "keyword" makes it a bit confusing. How about "This keyword is mandatory to identify that this package is a mediation policy type."?